jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsExtension.java

```diff
@@ -29,7 +29,7 @@ public class JaxrsExtension implements Extension {
 
     public static final String SUBSYSTEM_NAME = "jaxrs";
 
-    static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(4, 0, 0);
+    static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(3, 1, 0);
```

Comment:
Why are we downgrading the model version? The model version and the schema version are managed differently and do not need to be the same value.

Reply:
I don't understand why we need to use a different version. Why do we have to use 4.0.0 here ? It seems using the same version with the schema  makes more sense.  Th another reason I changed version here is I saw the transformer register here seems not correct :  
https://github.com/wildfly/wildfly/blob/main/jaxrs/src/main/java/org/jboss/as/jaxrs/ResteasyExtensionTransformerRegistration.java#L35-L39

Reply:
It turns out this model version is already used in the previous release and we can't do things to align with schema version. I already changed back and upgraded the model version to 4.1.0 for schema version 3.1.0.